internal/driver/mobile/app/android.c

```diff
@@ -70,10 +70,12 @@ static int main_running = 0;
 void processOnResume(ANativeActivity *activity) {
 	JNIEnv* env = activity->env;
 	setCurrentContext(activity->vm, (*env)->NewGlobalRef(env, activity->clazz));
-
-    onResume(activity);
```

Comment:
This cannot be right can it?
It looks like you have moved 3 of the callbacks from Go to C but deleted this one?